docs/docs/quickstart/lido.mdx

```diff
@@ -104,7 +104,30 @@ meantime.
 ## Managing Your Node Operator
 
 Once your validator keys are deposited and your node is running, you can manage your Node Operator using the Lido
-CSM Widget. This interface allows you to:
+CSM Widget and Sedge commands.
+
+### Checking Node Operator Status
```

Comment:
Please add more information regarding the reward address. It should be clear to the user what address to use as the command argument.